docker/README.md

```diff
@@ -8,10 +8,10 @@ We build, assemble, and test our artifacts on docker containers. All of our pipe
 
 ### Build CI Runner Docker Image from Dockerfile
 
-If you only want to build the docker image for either x64 or arm64, run this on a x64 or arm64 host respectively:
+To build the docker image for either x64 or arm64, run docker on a x64 or arm64 host respectively within the `opensearch-build/docker/ci` folder:
```

Comment:
```suggestion
To build the docker image for either x64 or arm64, run the below command on a x64 or arm64 host respectively within the `opensearch-build/docker/ci` folder:
```

Reply:
> Suggested change
> To build the docker image for either x64 or arm64, run docker on a x64 or arm64 host respectively within the `opensearch-build/docker/ci` folder:
> To build the docker image for either x64 or arm64, run the below command on a x64 or arm64 host respectively within the `opensearch-build/docker/ci` folder:

Done,could you check it please when you have free time?